include/mbedtls/chacha20.h

```diff
@@ -197,6 +198,20 @@ int mbedtls_chacha20_crypt(const unsigned char key[32],
                            const unsigned char *input,
                            unsigned char *output);
 
+/**
+ *  \brief          This function gets the value of CHACHA20_NONCE_8_BYTES
+ * 
+ *  \return         Nonce length
+*/
+int mbedtls_chacha20_get_nonce_length(void);
+
+/**
+ *  \brief          This function sets the value of CHACHA20_NONCE_8_BYTES
+ * 
+ *  \param length   Nonce length 
+*/
+void mbedtls_chacha20_set_nonce_length(uint8_t length);
```

Comment:
Check if you can leverage iv_size in mbedtls_cipher_context_t instead of these two implementations.